src/Microsoft.DotNet.ExecutablePackageObtainer/ToolConfiguration.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.DotNet.ExecutablePackageObtainer
+{
+    public class ToolConfiguration
+    {
+        public ToolConfiguration(
+            string commandName,
+            string toolAssemblyEntryPoint)
+        {
+            if (string.IsNullOrWhiteSpace(commandName))
```

Comment:
This is domain type of ToolConfiguration, I don't want to re use generated dto it is in adapter layer